stuff/pipelines/client_contest.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python3
+
+'''Processing contest messages.'''
+
+import argparse
+import logging
+import os
+import sys
+import json
+from typing import Any, List, Optional, Tuple
+import MySQLdb
+import MySQLdb.cursors
+import pika
+from verification_code import generate_code
+import rabbitmq_connection
+
+import omegaup.api   # pylint: disable=import-error
+
+sys.path.insert(
+    0,
+    os.path.join(
+        os.path.dirname(os.path.dirname(os.path.realpath(__file__))), "."))
+import lib.db   # pylint: disable=wrong-import-position
+import lib.logs  # pylint: disable=wrong-import-position
+
+
+def certificate_contests_receive_messages(
+        cur: MySQLdb.cursors.BaseCursor,
+        dbconn: MySQLdb.connections.Connection,
+        channel: pika.adapters.blocking_connection.BlockingChannel) -> None:
+    '''Receive contest messages'''
+    result = channel.queue_declare(queue='', exclusive=True)
+    queue_name = result.method.queue
+    assert queue_name is not None
+    channel.queue_bind(
+        exchange='certificates',
+        queue=queue_name,
+        routing_key="ContestQueue")
+    logging.info('[*] waiting for the messages')
+
+    def certificate_contests_callback(
+            _channel: pika.adapters.blocking_connection.BlockingChannel,
+            _method: pika.spec.Basic.Deliver,
+            _properties: pika.spec.BasicProperties,
+            body: bytes) -> None:
+        data = json.loads(body.decode())
+        client = omegaup.api.Client(
+            api_token='01fef0b0d78f56be29d42a25dacc69b743158039')
+        scoreboard = client.contest.scoreboard(contest_alias=data['alias'],
+                                               token=data['scoreboard_url'])
+        ranking = scoreboard['ranking']
+        certificates: List[Tuple[str, Any, str, Optional[Any], Any]] = []
```

Comment:
```suggestion
        certificates: List[Tuple[str, int, str, Optional[str], str]] = []
```

hay que evitar el uso de `Any`.

Reply:
Listo